src/jdk.zipfs/share/classes/jdk/nio/zipfs/ZipFileStore.java

```diff
@@ -35,11 +35,12 @@
 import java.nio.file.attribute.FileOwnerAttributeView;
 import java.nio.file.attribute.FileStoreAttributeView;
 import java.nio.file.attribute.PosixFileAttributeView;
+import java.util.Objects;
 
 /**
  * @author Xueming Shen, Rajendra Gutupalli, Jaya Hangal
  */
-class ZipFileStore extends FileStore {
+final class ZipFileStore extends FileStore {
```

Comment:
This should be OK but might suggest adding a release note

Reply:
> This should be OK but might suggest adding a release note

Do we need to do that for a package-private class? Maybe for internal use?

Reply:
This is a implementation class in a non-exported package so it's okay.

Reply:
> This is a implementation class in a non-exported package so it's okay.

Fair point